src/strings/unicode_ops.c

```diff
@@ -106,7 +106,7 @@ MVMint64 MVM_unicode_string_compare
 /* Looks up a codepoint by name. Lazily constructs a hash. */
 MVMGrapheme32 MVM_unicode_lookup_by_name(MVMThreadContext *tc, MVMString *name) {
     MVMuint64 size;
-    char *cname = MVM_string_ascii_encode(tc, MVM_string_uc(tc, name), &size, 0);
+    char *cname = MVM_string_ascii_encode(tc, name, &size, 0);
```

Comment:
Hm, doesn't this lose us case-insensitivity on codepoint names? Is this an intentional consequence? (Or is it just that we never notice since we don't expose the old op in Perl 6 space any more and use the new one for all `\c[...]` things?)